packages/quantic/force-app/examples/main/lwc/exampleQuanticBreadcrumbManager/exampleQuanticBreadcrumbManager.html

```diff
@@ -0,0 +1,37 @@
+<template>
+  <c-example-layout
+    title={pageTitle}
+    description={pageDescription}
+    show-preview={isConfigured}>
+  
+    <div slot="configuration">
+      <c-configurator options={options} ontryitnow={handleTryItNow}>
+      </c-configurator>
+    </div>
+
+    <c-quantic-search-interface slot="preview" engine-id={engineId}>
+      <div class="slds-grid slds-gutters_direct slds-wrap main slds-grid_align-center">
+        <div class="slds-col slds-order_2 slds-large-order_1 slds-size_1-of-1 slds-large-size_6-of-12">
+          <c-quantic-facet-manager engine-id={engineId}>
+            <c-quantic-facet display-values-as="link" field="filetype" label="File Type" engine-id={engineId}></c-quantic-facet>
+            <c-quantic-numeric-facet field="ytlikecount" label="Youtube Likes" with-input="integer" engine-id={engineId}></c-quantic-numeric-facet>
+            <c-quantic-timeframe-facet engine-id={engineId} field="date" label="Date">
+              <c-quantic-timeframe unit="week"></c-quantic-timeframe>
+              <c-quantic-timeframe unit="month"></c-quantic-timeframe>
+              <c-quantic-timeframe amount="6" unit="month"></c-quantic-timeframe>
+              <c-quantic-timeframe unit="year"></c-quantic-timeframe>
```

Comment:
It would be nice to have a timeframe with a custom label so we can validate that the breadcrumb shows the same label as the facet.

    <c-quantic-timeframe amount="10" unit="year" label="Past decade"></c-quantic-timeframe>